membership_policy.md

```diff
@@ -0,0 +1,125 @@
+  * Have your sponsoring reviewers reply confirmation of sponsorship: +1
+  * Once your sponsors have responded, your request will be reviewed by KubeVirt project approvers. Any missing information will be requested.
+
+## Responsibilities and privileges
+
+  * Responsive to issues and PRs assigned to them
+  * Responsive to mentions
+  * Active owner of code they have contributed (unless ownership is explicitly transferred)
+  * Code is well tested
+  * Tests consistently pass
+  * Addresses bugs or issues discovered after code is accepted
+  * Members can do /lgtm on open PRs.
+  * They can be assigned to issues and PRs, and people can ask members for reviews with a /cc @username.
+  * Tests can be run against their PRs automatically. No /ok-to-test needed.
+  * Members can do /ok-to-test for PRs that have a needs-ok-to-test label, and use commands like /close to close PRs as well.
+
+!! note
+Members who frequently contribute code are expected to proactively perform code reviews and work towards becoming a primary reviewer.
+
+## Reviewer
+
+Reviewers are trusted to review code for quality and correctness on some part of the project. They are knowledgeable about both the codebase and software engineering principles.
+Defined by: reviewers entry in an OWNERS file of the KubeVirt project.
+Reviewer status is scoped to a part of the codebase.
+Note: Acceptance of code contributions requires at least one approver in addition to the assigned reviewers.
+
+## Requirements
+
+The following apply to the part of codebase for which one would be a reviewer in the [OWNERS](https://github.com/kubevirt/kubevirt/blob/master/OWNERS_ALIASES) file.
+
+  * Member for at least 3 months
+  * Primary reviewer for at least 5 PRs to the codebase
+  * Reviewed or merged at least 20 substantial PRs to the codebase
+  * Knowledgeable about the codebase
+  * Sponsored by an approver
+    * With no objections from other approvers
+    * Done through PR to update the OWNERS file
+  * May either be self-nominate or nominated by an approver
+
+### Responsibilities and privileges
+
+  * The following apply to the part of codebase for which one would be a reviewer in the [OWNERS](https://github.com/kubevirt/kubevirt/blob/master/OWNERS_ALIASES) file.
+  * Tests are automatically run for PullRequests from members of the KubeVirt GitHub organization
+  * Code reviewer status may be a precondition to accepting large code contributions
+  * Responsible for project quality control via code reviews
+  * Focus on code quality and correctness, including testing and factoring
+  * May also review for more holistic issues, but not a requirement
+  * Expected to be responsive to review requests as per community expectations
+  * Assigned PRs to review related to project
+  * Assigned test bugs related to project
+  * Granted "read access" to KubeVirt repo
+  * May get a badge on PR and issue comments
+
+## Approver
```

Comment:
@dankenigsberg writes ...
```
I see that we have "emeritus_approvers". How an approver becomes one? What does it mean?
```

@stu-gott writes ...
```
That's a good point. Those in that category used to be active members of the project, but have moved on for various reasons. @rmohr@redhat.com might have insight as to what effect being in emeritus status has for the purposes of the owners file ("git blame" indicates he's edited that section).
```

@rmohr writes ...
```
prow will suggest contributors once a PR went through inital review and looks too, to add an approver to the PR. It will suggest people based on the owners file. I therefore clean up the owners file from time to time, so that the suggestions make sense.

Approvers which left the project or don't contribute often enough anymore are moved to the emeritus section for two reasons:
* current approvers can see who may have insight about things which were done in the past
* to honor their past contributions
```